README.md

```diff
@@ -81,18 +82,9 @@ Current state: {'total': 133.105, 'power': 108.223577, 'current': 0.54463, 'volt
 
 ## Plug-specific commands
 
-At the moment only switching the state of the LED is implemented.
-**Feel free to submit patches as pull requests for further features!**
-### Controlling the LED
-
-`led` command can be used to control whether the LED light on front of the plug is on or off.
+At the moment switching LED state and brightness (for dimmable switches) is suported.
 
-```
-$ pyhs100 --plug led
-LED state: False
-$ pyhs100 --plug led 1
-Turning led to True
-```
+**Feel free to submit patches as pull requests for further features!**
```

Comment:
Hmm, why move this below to API section?

Reply:
I wasn't 100% sure.  I was trying to be consistent with how the "Bulb" directions were below it.  Previously, for the plug section, it said it only supported LED On/OFF, but since it also now supports "brightness" and given the "## Bulb-specific commands" were empty, with all commands below the "## Bulb-specific API" section, I figured maybe it made sense for consistency to do the same for "Plug".  Again, it's certainly possible I'm reading it wrong.

Reply:
Ah, I see what you mean now.  Will adjust accordingly.

Reply:
Any news on this?

Reply:
Sorry, work has taken over lately.  I will try and tweak and submit tonight.  Thanks.